defaults/aws_config.yaml

```diff
@@ -1,4 +1,5 @@
-instance_provision: 'spot_low_price'
+instance_provision: 'spot_duration'
```

Comment:
why isn't it "spot"?

Reply:
I put "spot_duration" because of it's real type, by code it's the same as "spot" and hidden from user